### DISCUSSION TITLE
I can't apply for developer pack

### DISCUSSION THREAD

**Discussion by civanozmen** [community/community#128665] [GitHub Education]
### Hi everyone,

I have problems with my student developer pack application. For years I've been used my student transcript (dated, digital png) for the application. But this time it rejected my transcript. So i've tried it with a real copy of my transcript (also dated but not digital, real A4 copy of my transcript). But this time probably the problem is my camera quality, I wanna get the pack but whatever i've tried it doesn't matter I always get a error. Can GitHub support help me with my problem?

**Comment by yilmazemrecom** [comment]
You can open support from here. https://docs.github.com/en/support/contacting-github-support
Also, don't forget to use the email address your school gave you for GitHub Edu.

**Reply by samus-aran** [reply]
All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

**Comment by samus-aran** [comment]
Hi @civanozmen 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.

1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
5. Alternatively you may use another mobile device or computer.

Also suggest that you take a photo of much as the text as possible rather than the outer edges if that will help with the quality.
